sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Filtering/DerivedMetric.cs

```diff
@@ -201,7 +201,9 @@ private void CreateProjection()
                             MethodInfo? parseMethod = typeof(TimeSpan).GetMethod("Parse", new[] { typeof(string) });
                             fieldExpression = Expression.Call(parseMethod!, fieldExpression);
                         }
-                        fieldExpression = Expression.Property(fieldExpression, "TotalMilliseconds");
+
+                        var totalMillisecondsProperty = typeof(TimeSpan).GetProperty(nameof(TimeSpan.TotalMilliseconds))!;
+                        fieldExpression = Expression.Property(fieldExpression, totalMillisecondsProperty);
```

Comment:
This is safe because we've already identified that we're working with a `TimeSpan` at line 197.
"TotalMilliseconds" is a known public property, so we don't need an `Expression` to lookup this property by name. The new code uses `GetProperty()` to provide a `PropertyInfo` to the Expression.